crates/bevy_render/src/camera/projection.rs

```diff
@@ -369,17 +369,6 @@ pub struct OrthographicProjection {
     ///
     /// Defaults to `ScalingMode::WindowSize(1.0)`
     pub scaling_mode: ScalingMode,
-    /// Scales the projection.
-    ///
-    /// As scale increases, the apparent size of objects decreases, and vice versa.
-    ///
-    /// Note: scaling can be set by [`scaling_mode`](Self::scaling_mode) as well.
-    /// This parameter scales on top of that.
```

Comment:
This makes it pretty easy to scale multiple variants of `ScalingMode` since you can just modify a single `f32`. Are we at all concerned about hurting ergonomics?

Reply:
Not particularly; projects will only need to handle this once, and the majority of projects will have a single scaling mode.

Reply:
I've also run into a few people now who miss the existence of `ScalingMode` because of `scale`, leading them to ask questions. In this way, I think it's hurting them by not just making them think up front about how the camera should be configured.